vue/src/components/siteObservations/EvaluationImages.vue

```diff
@@ -184,6 +184,24 @@ const imageDownloadDialog = ref(false);
         <b>{{ siteObservation.imageCounts.WV.loaded !== 0 ?Math.max(siteObservation.imageCounts.WV.loaded - siteObservation.imageCounts.WV.total, 0) : '-' }}</b>
       </v-col>
     </v-row>
+    <v-row
+      dense
+      justify="center"
+      align="center"
+    >
+      <v-col cols="3">
+        <b>PL</b>
+      </v-col>
+      <v-col cols="3">
+        <b>{{ siteObservation.imageCounts.PL.loaded }}</b>
+      </v-col>
+      <v-col cols="3">
+        <b>{{ siteObservation.imageCounts.PL.total }}</b>
+      </v-col>
+      <v-col cols="3">
+        <b>{{ siteObservation.imageCounts.PL.loaded !== 0 ?Math.max(siteObservation.imageCounts.PL.loaded - siteObservation.imageCounts.PL.total, 0) : '-' }}</b>
```

Comment:
Minor and I think initially my fault.
```suggestion
        <b>{{ siteObservation.imageCounts.PL.loaded !== 0 ? Math.max(siteObservation.imageCounts.PL.loaded - siteObservation.imageCounts.PL.total, 0) : '-' }}</b>
```